### PR TITLE
10180 move all incubator modules of elico8 to public GitHub elico web css

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: web_calendar_css_customization/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Web Calendar CSS Customization
+==============================
+
+This module extends the functionality of web calendar css customization.
```
Comment (elicoidal) [inline web_calendar_css_customization/README.rst:9]: @uynil Can you give more precision about this module feature?

File: web_calendar_css_customization/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Web/Css Customization",
+    "version": "1.0",
```
Comment (siyuancn) [inline web_calendar_css_customization/__openerp__.py]: wrong version

**Comment by siyuancn** [comment]
Please add the License file

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

**Comment by elicoidal** [comment]
I dont really see the value of this module (1 line of code and not clear purpose)
Please cancel the PR

**Comment by Rona111** [comment]
@elicoidal ok,I have cancled.

**Pull request closed without merging**